spectral_cube/spectral_axis.py

```diff
@@ -72,6 +72,14 @@ def wcs_unit_scale(unit):
         if wu.is_equivalent(unit):
             return wu.to(unit)
 
+def parse_phys_type(unit):
+    '''
+    As of astropy 4.3.dev1499+g5b09f9dd9, the physical type of a speed if "speed".
```

Comment:
is this supposed to say `is "speed"`, not  `if "speed"`?

Reply:
and... isn't the problem that it was `speed`, and is now `speed/velocity`?

Reply:
Oh. I did a replace all... will fix.